### DISCUSSION TITLE
No ability to backdate an on/off hire if the asset is currently on hire

### DISCUSSION THREAD

**Discussion by BigLifts** [stayonhire/feedback#20] [Feedback / Issues]
1. If an asset is currently on hire (purple job)
2. The earlier job on/off hire (yellow job) confirmation can't be completed 
![Screenshot 2023-07-18 at 3 03 08 pm](https://github.com/stayonhire/feedback/assets/139850205/7605d403-7473-465f-8b93-00587d82e4bf)

3. If you try, an error message will appear
![Screenshot 2023-07-18 at 3 04 15 pm](https://github.com/stayonhire/feedback/assets/139850205/06514c40-73f2-4d1d-89e8-7e09c4d2a146)

**Comment by dylanhru** [comment]
Older hires are now labelled at PRE-HIRE and can be backdated to start and finish.
<img width="795" alt="Screenshot 2023-08-03 at 8 11 53 pm" src="https://github.com/stayonhire/feedback/assets/34927957/1c585b07-b1a8-48be-aa1d-191c3f0c4b02">
